.github/workflows/ci.yaml

```diff
@@ -10,7 +10,7 @@ on:
       - 'release-*'
 env:
   GO_VERSION: 1.19
-  K8S_VERSION: 1.24.2
+  K8S_VERSION: 1.25.0
```

Comment:
this cannot be changed yet because there's no cluster available for `setup-envtest`.

Reply:
it triggers the "CI / Build and check..." error: `unable to fetch checksum for requested version: unable fetch metadata for kubebuilder-tools-1.25.0-linux-amd64.tar.gz -- got status "404 Not Found" from GCS`